README.md

```diff
@@ -1,31 +1,33 @@
-> **This is a "cracked" version of a popular Minecraft launcher that lets you play the game without a Mojang account.**
->
-> This software is not related to MultiMC developers and provided without any warranty. Please don't bomb MultiMC developers if something gets wrong using this launcher.
+# **This is a "cracked" version of a popular Minecraft launcher that lets you play the game without a Mojang account.**
+
+# This software is not related to MultiMC developers and provided without any warranty. Please don't bomb MultiMC developers if something gets wrong using this launcher.
 
 ## Pre-built binaries:
 - Windows / Linux: https://nightly.link/AfoninZ/MultiMC5-Cracked/workflows/main/develop
 - MacOS: https://drive.google.com/file/d/1QKjeghZecHH9foduy6dKEmpAH9AHbRTA/view?usp=sharing
 
 ## How to install and use
-1. Download pre-built binaries for your system
-2. Unpack them in your desired directory
-3. Launch the MultiMC
-4. Go to account settings, and you will be requested to use email and password
-5. As your email use your username, and then type a random password
-6. Save it
-7. Now enjoy MultiMC
+1. Download pre-built binaries for your system.
+2. Unpack them in your desired directory.
+3. Launch `DevLauncher`.
+4. Go to account settings, and you will be requested to use email and password.
+5. As your email use your username, and then type a random password or use ElyBy account.
+6. Save it.
+7. Now enjoy the Launcher.
+
+In case if you are using MacOS/Linux then additional step is to make a script `DevLauncher` executable by using the command `chmod +x DevLauncher` in the terminal
 
-In case if you are using MacOS/Linux then additional step is to make a script `MultiMC` executable by using the command `chmod +x MultiMC` in the terminal
+To remove the ugly "localhost:nnnn" or "ElyBy", add `-Dauthlibinjector.noShowServerName` to the java arguments.
```

Comment:
Please remove word ugly, be more polite to information

Reply:
Done.